### PR TITLE
api-endpoint-hotfix

### PR DESCRIPTION
**Pull request by swjeong0825**
1. remove duplicated endpoint.
2. preserve naming convention
3. fix error about tag filter

### CODE REVIEW

File: frontend/src/pages/JobFeed/components/TagSelect.tsx [hunk 1]
```diff
@@ -19,7 +19,12 @@ type TagSelectProps = {
 export default function TagSelect(props: TagSelectProps) {
   const { onChange, value } = props
   const theme = useTheme()
-  const [{ data }] = useAxios('http://localhost:5000/tags')
+  const [{ data }] = useAxios({
+    url: 'http://localhost:5000/tag',
```
Comment (Danny123234) [inline frontend/src/pages/JobFeed/components/TagSelect.tsx:23]: based on what stu said today in the meeting, the url should be just `/tag` you dont need the host and port stuff

**Review by Danny123234 — CHANGES_REQUESTED** [review]

File: frontend/src/pages/JobFeed/components/TagSelect.tsx (hunk 1 above)
Comment (swjeong0825) [reply frontend/src/pages/JobFeed/components/TagSelect.tsx]: Changed. 
I used the "grep -Rl "localhost" . --exclude-dir={node_modules,build,public}" command, so there should not be any localhost or 127.0.0.1 stuff anymore

**Review by Danny123234 — APPROVED** [review]
Ya it looks much cleaner now!

**Review by rdonati — APPROVED** [review]
👍

**Pull request merged**